src/main/target/common_post.h

```diff
@@ -323,7 +323,7 @@
 #endif
 #endif
 
-#if (defined(USE_FLASH_W25M512) || defined(USE_FLASH_W25Q128FV)) && !defined(USE_FLASH_M25P16)
+#if (defined(USE_FLASH_W25M512) || defined(USE_FLASH_W25Q128FV) || defined(USE_FLASH_PY25Q256HB)) && !defined(USE_FLASH_M25P16)
```

Comment:
It seems `USE_FLASH_PY25Q256HB` does nothing but result in `USE_FLASH_M25P16` being defined.

Perhaps we can simplify all variants that follow the `USE_FLASH_M25P16` format and remove any individual flash chip USE definitions to simplify the generic nature of this layout / format.

Reply:
Yes that's correct.

Makes total sense. I was just following the current scheme

Reply:
i've noticed this as well and yet never looked at the code.  e.x. M25P16 will get a W25Q128FV to work, but i thought maybe the latter would add some efficiency.  so are ya'll sating that in fact, that there is no efficiency added by W25Q128FV code?

Reply:
Only if you use QSPI, otherwise it uses the exact same code base.